Consul/Client_GetRequests.cs

```diff
@@ -171,6 +171,27 @@ protected override void ApplyOptions(ConsulClientConfiguration clientConfig)
             {
                 Params["near"] = Options.Near;
             }
+
+            if (Options.UseCache)
+            {
+                Params["cached"] = string.Empty;
+                var cacheControl = new List<string>();
+
+                if (Options.MaxAge.HasValue && Options.MaxAge > TimeSpan.Zero)
```

Comment:
Argument validation on the client side is not necessary, it is up to a consul agent to decide what to do with invalid arguments. 
```suggestion
                if (Options.MaxAge.HasValue)
```